scripts/db_migrate.sh

```diff
@@ -19,6 +18,12 @@ read -rp "Enter (FROM) database name: " DB_NAME
 read -rp "Enter (TO) port number:" DB_TO_PORT
 read -rp "Enter (TO) database name: " DB_TO_NAME
 
+# Given the output of "docker ps" and a port, can find a matching line of:
+# 6eabb950ac3f   mariadb:10.2  "docker-entrypoint.s…" 2 weeks ago Up 30 hours 0.0.0.0:33102->3306/tcp sourcetoad_mariadb102
+# and extract "sourcetoad_mariadb102"
+CONTAINER_NAME_FROM=$(docker ps | grep "$DB_PORT" | rev | cut -d" " -f1 | rev)
+CONTAINER_NAME_TO=$(docker ps | grep "$DB_TO_PORT" | rev | cut -d" " -f1 | rev)
```

Comment:
You can make this a little more readable by having docker give you only the columns you want
```shell
docker ps --format '{{.Names}} {{.Ports}}' | grep "$DB_PORT" | cut -d ' ' -f1
```

Reply:
nice that makes it feel much less dirty thanks!